app/src/components/content/LanguageModal.vue

```diff
@@ -0,0 +1,56 @@
+<script setup lang="ts">
+import { db, DocType, type LanguageDto } from "luminary-shared";
+import LButton from "../button/LButton.vue";
+import { CheckCircleIcon } from "@heroicons/vue/20/solid";
+import { appLanguageIdAsRef } from "@/globalConfig";
+
+type Props = {
+    isVisible: boolean;
+};
+defineProps<Props>();
+
+const languages = db.whereTypeAsRef<LanguageDto[]>(DocType.Language, []);
+
+const emit = defineEmits(["close"]);
+
+const setLanguage = (language: LanguageDto) => {
+    appLanguageIdAsRef.value = language._id;
+    emit("close");
+};
+</script>
+
+<template>
+    <div
+        v-if="isVisible"
+        class="fixed inset-0 z-50 flex items-center justify-center bg-gray-800 bg-opacity-75"
+    >
+        <div class="w-11/12 rounded-lg bg-white p-6 shadow-lg sm:w-96 dark:bg-zinc-900">
+            <h2 class="mb-4 text-lg font-semibold">Select Language</h2>
+            <ul class="divide-y divide-gray-200">
+                <li
```

Comment:
Currently while pressing Tab is doesn't focus these buttons, which is what I would expect. This is most easiest solved by using an actual `button` element. Then the `ul` can be changed to a `div` for example.